### ISSUE TITLE
Error during commit

### ISSUE THREAD

**Issue by sprinz11** [rbtl-fs24/md-02-assignments-sprinz11#1]
@larnsce 

I'm running into a problem when I try to commit my changes with the following error message:

![grafik](https://github.com/rbtl-fs24/md-02-assignments-sprinz11/assets/114858201/97ba6b6d-702a-4e3c-867a-1fa9cd95a3de)

How do I fix that?

**Comment by sprinz11** [comment]
I think I made it work now by running this in the console:

`library(usethis)`
`use_git_config(user.name = "sprinz11", user.email = "sprinz@student.ethz.ch")`

**Comment by larnsce** [comment]
Yes, well done for figuring it out yourself. This should have been sorted when your worked through the [git configuration](https://rbtl-fs24.github.io/website/assignments/md-02/am-02-1-github.html). I am surprised that you had to do it again. Hopefully it's fine for future assignments now.

**Comment by larnsce** [comment]
You can close this issue now.

**Comment by sprinz11** [comment]
Okay, I hope so, too. :)

**Comment by sprinz11** [comment]
Unfortunately, it is not fixed yet. I still get the same error message.

**Comment by larnsce** [comment]
Thanks for letting me know. Could you come to the student hours next Tuesday? We could also do a Zoom call.

**Comment by larnsce** [comment]
Closing this out: the rbtl-fs24 course ended in 2024. I never got a follow-up here, so I'm assuming this is no longer active. Apologies for the late reply, and reopen if you'd still like help.
